app/controllers/api/messaging.php

```diff
@@ -27,9 +28,9 @@
 use Utopia\Validator\WhiteList;
 
 App::post('/v1/messaging/providers/mailgun')
-    ->desc('Create Mailgun Provider')
+    ->desc('Create Mailgun provider')
     ->groups(['api', 'messaging'])
-    ->label('audits.event', 'providers.create')
+    ->label('audits.event', 'provider.create')
     ->label('audits.resource', 'providers/{response.$id}')
```

Comment:
Audits resources should be singular form as well, like `provider/{response.$id}`

Reply:
done this